README.md

```diff
@@ -76,6 +77,10 @@ export default class App extends Component {
         selected: selected.filter(t => t.value !== tag.value)
       })
     }
+    
+    const search = (tag, input, selectedTags) => {
+      tag.label.toLowerCase().includes(input.toLowerCase()) && !selected.includes(tag)
```

Comment:
Your arg name is `selectedTags` but you're referencing `selected`. You're also not passing this function into the component below.